query-cloudtrail.sh

```diff
@@ -1,13 +1,22 @@
 #!/bin/bash
+
 set -eu
-echo
-echo "Date:         $(date)"
-echo "S3 bucket:    ${BUCKET}"
-echo
-set -x
-aws cloudtrail lookup-events --region us-gov-west-1 --lookup-attributes AttributeKey=ResourceName,AttributeValue=${BUCKET} \
+
+for bucket in $(aws s3 ls | grep federalist | awk '{print $3}'); do
```

Comment:
Not all of the Federalist user sites are using the `federalist-...` bucket naming convention since the the creation of the Federalist S3 broker came after Federalist's migration to the one site, one bucket paradigm.  Would installing and logging into Federalist's CF org to run and parse `cf services` be much of a lift Concourse?

Reply:
@apburnes that would be a good amount of extra complication.  If we can say for sure that the non `federalist-*` buckets are all known, and that this list isn't going to grow, then I'd rather hard-code them into this repo.  Would that work?  If so, can you provide that list?

Reply:
@tammersaleh sounds good. We can most definitely get the list.  👍